cms/envs/bok_choy.py

```diff
@@ -93,6 +93,8 @@
 FEATURES['ENABLE_MOBILE_REST_API'] = True  # Enable video bumper in Studio
 FEATURES['ENABLE_VIDEO_BUMPER'] = True  # Enable video bumper in Studio settings
 
+FEATURES['ENABLE_ENROLLMENT_TRACK_USER_PARTITION'] = True
```

Comment:
Note to reviewers: feature flag was introduced because the next story will add the ability to set component visibility based on this new partition. @sstack22 doesn't want course authors seeing this functionality before the other parts (including the ability to segment discussions) are present.

Hopefully this feature flag will be deleted once everything is complete, as our plan is to make the feature invisible for courses where only a single enrollment track exists.